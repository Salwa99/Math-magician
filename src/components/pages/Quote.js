import React from 'react';
// eslint-disable-next-line
const Quote = () => (

  <div className="quote">
    <p>&quot;Mathematics is the music of reason.&quot; James Joseph Sylvester </p>
  </div>
);
export default Quote;
