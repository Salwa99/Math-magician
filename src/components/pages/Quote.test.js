import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('renders when there is quote', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
