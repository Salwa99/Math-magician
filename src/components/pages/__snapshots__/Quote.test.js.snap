// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is quote 1`] = `
<div
  className="quote"
>
  <p>
    "Mathematics is the music of reason." James Joseph Sylvester 
  </p>
</div>
`;
